clientv3/integration/watch_test.go

```diff
@@ -678,7 +678,7 @@ func TestWatchErrConnClosed(t *testing.T) {
 	clus.TakeClient(0)
 
 	select {
-	case <-time.After(3 * time.Second):
+	case <-time.After(10 * time.Second):
```

Comment:
if dial timeout is 5 seconds, then 6 seconds should be enough? also can we simply reduce dial timeout?